java/runtime-dev/src/main/java/org/ray/spi/impl/MockLocalScheduler.java

```diff
@@ -96,8 +89,8 @@ public UniqueId generateTaskId(UniqueId driverId, UniqueId parentTaskId, int tas
   }
 
   @Override
-  public List<byte[]> wait(byte[][] objectIds, int timeoutMs, int numReturns) {
-    return store.wait(objectIds, timeoutMs, numReturns);
+  public <T> WaitResult<T> wait(List<RayObject<T>> waitFor, int numReturns, int timeout) {
+    throw new RuntimeException("Not implemented here.");
```

Comment:
It shouldn't throw exception here. But I think `MockLocalScheduler` is already broken, let's fix it later.